views/layouts/main.tt

```diff
@@ -9,7 +9,7 @@
     <link rel="stylesheet" href="[% url.css %]/external.css?v=7">
     <link rel="stylesheet" href="[% url.css %]/general.css?v=8">
   </head>
-  <body class="[% body_class || 'page' %]"[% IF csrf_token %] data-csrf="[%- csrf_token -%]"[% END; IF layout OR layout_obj; %] data-layout-identifier="[% layout.identifier || layout_obj.identifier %]"[% END %]>
+  <body class="[% body_class || 'page' %]"[% IF csrf_token %] data-csrf="[%- csrf_token -%]"[% END; IF layout OR layout_obj; %] data-layout-identifier="[% layout.identifier || layout_obj.identifier %]"[% END %][% IF actions; %][% IF actions.update_record.defined %] data-update-record="[% actions.update_record %]"[% END %][% END; %]>
```

Comment:
I'm wondering whether the actions object can be passed into the body data as its full object rather than splitting out a particular action?